docs/Developers/Git_Basics.rst

```diff
@@ -44,15 +44,25 @@ To do this, you can use these to clone the coala/coala-bears repositories:
 
 ::
 
-    $ git clone https://github.com/coala/coala
+    $ git clone -o upstream https://github.com/coala/coala
 
 or
 
 ::
 
-    $ git clone https://github.com/coala/coala-bears
+    $ git clone -o upstream https://github.com/coala/coala-bears
+
+.. note::
+
+    ``-o upstream`` sets the remote name of the original coala/coala-bears
```

Comment:
>coala/coala-bears is consistently used throughout the document. I'm following the existing convention of how to refer to coala/coala-bears

I see, actually to me it seems like "coala/coala-bears" refers to https://github.com/coala/coala-bears, but as you've changed `repository` to `repositories` then it should refer to both "coala/coala" and "coala/coala-bears".